src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -62,6 +63,29 @@ scope core
         struct Unrouted extends core::control::Frame [0x40000002]
         {
         }
+
+        scope auth
+        {
+            struct Resolve extends core::control::Frame [0x00000003]
```

Comment:
Prefer to start these `auth` frame type IDs at `[0x00000011]` instead please.

Reply:
ok